portal/src/locale-data/en.json

```diff
@@ -643,7 +644,7 @@
   "ThemeConfigurationWidget.dark-mode": "Dark Mode",
   "ThemeConfigurationWidget.theme-color-title": "Theme Color",
   "ThemeConfigurationWidget.app-logo-title": "App Logo",
-  "ThemeConfigurationWidget.app-logo-description": "It helps the end users to identify your app, and it is shown in all pages, emails, etc. JPEG, PNG and GIF are supported.",
+  "ThemeConfigurationWidget.app-logo-description": "It helps the end users to identify your app, and it is shown in all pages, emails, etc. JPEG, PNG and GIF are supported. \n (Size limit: 100 KB)",
```

Comment:
You can `use {br, react}` to insert a `<br>` element. So we do not need `white-space: pre-line;`